File: python/palindromes/js/palindrome.js
exports.palindrome = function(word) {
	// Make sure it's a string and all letters are lowercase.
	const stringify = String(word).toLowerCase()
	// Remove special characters and whitespace
	const forward = stringify.replace(/[^a-zA-Z0-9]/g, '')
	// Reverse the string
	const backwards = forward.split("").reverse().join("")

	return true ? forward === backwards : false

}

File: python/palindromes/python/palindrome.py
import re


def palindrome(word):
    # Make sure it's a string and all letters are lowercase.
    stringify = str(word).lower()
    # Remove special characters and whitespace
    forward = str(re.sub("[^A-Za-z0-9]+", "", stringify))
    # Reverse the string
    backwards = forward[::-1]

    return True if forward == backwards else False
